lua/gitlab/actions/discussions/winbar.lua

```diff
@@ -49,9 +49,17 @@ local function content()
   local resolvable_notes, resolved_notes = get_data(state.DISCUSSION_DATA.unlinked_discussions)
 
   local draft_notes = require("gitlab.actions.draft_notes")
-  local inline_draft_notes = List.new(state.DRAFT_NOTES):filter(draft_notes.has_position)
-  local unlinked_draft_notes = List.new(state.DRAFT_NOTES):filter(function(note)
-    return not draft_notes.has_position(note)
+  local inline_draft_notes, unlinked_draft_notes = List.new(state.DRAFT_NOTES):partition(function(note)
+    if note.discussion_id == "" then
```

Comment:
Small thing but since this is an early return, it's clearer in my opinion if you remove the `else` block. Meaning:

```lua
if some_cond then
   return some_result
end

-- Do other work

```

Reply:
Oh, sure. It's indeed clearer. Fixed.